package.json

```diff
@@ -60,6 +60,7 @@
     "component-type": "^1.2.1",
     "component-url": "^0.2.1",
     "debug": "^0.7.4",
+    "eslint-plugin-react": "^7.9.1",
```

Comment:
Should this be a dev dependency?

Reply:
We don't really need it but it's a peer dependency of `@segment/eslint-config` so I added it to silent the yarn warning message